model/jpa/src/main/java/org/keycloak/models/jpa/JpaClientProviderFactory.java

```diff
@@ -36,19 +37,22 @@
 
     private Set<String> clientSearchableAttributes = null;
 
+    private static final List<String> REQUIRED_SEARCHABLE_ATTRIBUTES = Arrays.asList(
+      "saml_idp_initiated_sso_url_name"
+    );
```

Comment:
I wonder whether we should extract the default searchable attributes and make it independent from the JPA provider. I know JPA is currently the only one currently using the searchable attributes option, but that might change...

Reply:
That's right, but I'd rather keep it as is for the sake of this PR. If this would be needed later on, let's find the proper place only then